boards/nordic/nrf54l15pdk/nrf54l15pdk_nrf54l15_cpuflpr.dts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+ #include "nrf54l15pdk_nrf54l15_cpuflpr_xip.dts"
+
+&cpuflpr_sram {
+	reg = <0x20028000 DT_SIZE_K(96)>;
+	ranges = <0x0 0x20028000 0x18000>;
+};
```

Comment:
This changes the RAM location, shouldn't the `cpuflpr_code_partition` be changed as well (from RRAM to SRAM?). Or am I missing something?

Reply:
It is not necessary. This is handled via the `XIP` Kconfig.